sgip_api.py

```diff
@@ -52,25 +53,25 @@ def login(username, password):
         return None
 
 
-def moer(token, ba, starttime=None, endtime=None):
+def moer(token, ba, starttime=None, endtime=None, version=None):
     url = 'https://sgipsignal.com/sgipmoer'
     headers = {'Authorization': 'Bearer {}'.format(token)}
     params = {'ba': ba}
     if starttime:
-        params.update({'starttime': starttime, 'endtime': endtime})
+        params.update({'starttime': starttime, 'endtime': endtime, 'version': version})

```

Comment:
you don't want to send a version of None.  so you should remove version from this line and have another if statement
if version:
       params['version'] = version

Reply:
same for forecast

Reply:
Corrected!